tests/foreman/api/test_host.py

```diff
@@ -1068,7 +1081,7 @@ def test_positive_read_enc_information(
     host_enc_parameters = host_enc_info['data']['parameters']
     assert host_enc_parameters['organization'] == module_org.name
     assert host_enc_parameters['location'] == module_location.name
-    assert host_enc_parameters['content_view'] == module_cv_with_puppet_module.name
+    assert host_enc_parameters['content_view'] == default_contentview.label
```

Comment:
Is there a `default_contentview.name`?  Keeping up with consistencies, that's all.

Reply:
i change it on purpose here :) 
`host_enc_parameters['content_view'] = Default_Content_View`
`default_contentview.label = Default_Content_View`
`default_contentview.name = Default Content View` 
so just easier to use `label` as e.g `replace(' ', '_')`